Lib/fontParts/base/font.py

```diff
@@ -806,33 +1085,46 @@ def getLayer(self, name):
         self._setFontInLayer(layer)
         return layer
 
-    def _getLayer(self, name, **kwargs):
-        """
+    def _getLayer(self, name: str, **kwargs: Any) -> BaseLayer:
+        r"""Get the named layer from the native font.
+
         This is the environment implementation of
-        :meth:`BaseFont.getLayer`. **name** will
-        be a :ref:`type-string`. It will have been
-        normalized with :func:`normalizers.normalizeLayerName`
-        and it will have been verified as an existing layer.
-        This must return an instance of :class:`BaseLayer`.
+        :meth:`BaseFont.getLayer`.
+
+        :param name: The name of the :class:`BaseLayer` subclass
+            instance to retrieve.
+        :param \**kwargs: Additional keyword arguments.
+        :return: The specified :class:`BaseLayer` subclass instance.
+            The value will be normalized with :func:`normalizers.normalizeLayerName`
+            and verified as an existing layer.
+
+        .. note::
+
+            Subclasses may override this method.
 
-        Subclasses may override this method.
         """
         for layer in self.layers:
             if layer.name == name:
                 return layer
+        else:
```

Comment:
This `else` feels wrong, there is no corresponding if. Should just be the `raise` from the next line

Reply:
Indeed. Fixed, and added corresponding `:raises:` statement to docstring.